mscore/pianoroll/pianoview.cpp

```diff
@@ -1865,12 +1865,12 @@ void PianoView::cutNotes()
 
 void PianoView::copyNotes()
       {
-      QString data = serializeSelectedNotes();
-      if (data.isEmpty())
+      QString dat = serializeSelectedNotes();
```

Comment:
How about renaming `dat` to something more meaningful, such as `copiedNotes`?

Reply:
done